### ISSUE TITLE
Placeholder text doesn't meet minimum contrast requirements

### ISSUE THREAD

**Issue by paulcpederson** [Esri/calcite-web#703]
Need to make the gray used for `:placeholder` text darker gray so that the contrast ratio exceeds the 4.5:1 guideline.
